### ISSUE TITLE
Question about 

### ISSUE THREAD

**Issue by Aditi138** [facebookresearch/GENRE#23]
Hey,

Similar to the GENRE model example of Document Retieval, is it possible to get entity linking from sentences without the pre-speciying the [START] and [END] tokens for multilingual text?

**Comment by nicola-decao** [comment]
No, it is not possible. We only address entity linking with mGENRE https://arxiv.org/pdf/2103.12528.pdf .

**Comment by Aditi138** [comment]
Thanks!

**Comment by nicola-decao** [comment]
@Aditi138  of course if you need a model that does that fine-tuning mGENRE should be better than learning do to this from scratch 😉
